mono/mini/tramp-arm64-gsharedvt.c

```diff
@@ -277,7 +277,6 @@ mono_arch_get_gsharedvt_trampoline (MonoTrampInfo **info, gboolean aot)
 	/* Allocate callee register area just below the callee area so it can be accessed from start_gsharedvt_call using negative offsets */
 	/* The + 8 is for alignment */
 	callee_reg_area_offset = 8;
-	callee_stack_area_offset = callee_reg_area_offset + (n_arg_regs * sizeof (target_mgreg_t));
```

Comment:
needed?

Reply:
Doesn't look that way and it looks like it is handled using a bigger offset (both n_arg_regs and n_arg_fregs).